GOVERNANCE.md

```diff
@@ -0,0 +1,129 @@
+
+Core Contributors are those with a history of consistent contributions, including but not limited to pull requests, project management, or support. They benefit from a certain amount of privileges, including:
+
+- Push access to the repository, except the `master` / `main` branches
+- Contributor role on the Yarn Discord channel
+- Ability to vote on project decisions
+
+Being core promoted core contributor doesn't mean the member is required to stay active. Inactive contributors may have voting rights put on hold until they return, but will always retain their status.
+
+#### Induction
+
+Candidates must be privately nominated by an existing core contributors, which will send the proposal to a steward. Stewards will hold a private vote open to any core contributor, and if the proposal passes with a significant majority (75%) the candidate will be reached to be offered the position.
+
+### II. Stewards
+
+Stewards establish the project vision, have a full control over the code and its assets, and assume the right to speak for the project in public. In practical terms, stewards' privileges include:
+
+- Access to the @yarnpkg Twitter account
+- Administration privileges on the Yarnpkg GitHub org
+- Administration privileges on the Yarnpkg Discord server
+- Publish access to the `yarn` npm package (which we don't plan to use much)
+- Administration access to the `yarnpkg.com` accounts (Netlify, Cloudflare, ...)
+- Ability to veto votes and resolve voting deadlocks
+- Define project direction and planning
+- Ability to decide on moderation decisions
+
+In the event a steward becomes incapacitated, they are expected to leave keys to a trustee that will transmit the rights to an new steward appointed by the core contributors through a vote.
+
+#### Induction
+
+New stewards will be added based on a unanimous vote by the existing stewards - in the event that one of them is unreachable, the decision will be deferred until all stewards have voted. While discussion and approval will be generally done in private, stewards will hold an advisory poll open to core contributors.
+
+## Current Members
+
+### Stewards
+
+- Maël Nison - [arcanis](https://github.com/arcanis)
+
+### Core Contributors
+
+- Bram Gotink - [bgotink](https://github.com/bgotink)
+- Daniel Almaguer - [deini](https://github.com/deini)
+- Haroen Viaene - [haroenv](https://github.com/haroenv)
+- Kristoffer K. - [merceyz](https://github.com/merceyz)
+- Marc-Antoine - [embraser01](https://github.com/embraser01)
+- Paul Soporan - [paul-soporan](https://github.com/paul-soporan)
+- Sebastian Silbermann - [eps1lon](https://github.com/eps1lon)
+- Victor Vlasenko - [larixer](https://github.com/larixer)
+- Will Griffiths - [willgriffiths](https://github.com/willgriffiths)
+
+## Project direction and planning
+
+Project direction and planning is a shared responsibility amongst members. Stewards are responsible for dictating high level goals and scope of the project that should be adhered to.
```

Comment:
```suggestion
Project direction and planning is a shared responsibility amongst members. Stewards are responsible for defining high level goals and scope of the project that should be adhered to.
```

"Dictating" sounds like you're telling people what they should do and has, for me personally, a rather negative connotation.

Reply:
Much better indeed 👍